taskfiles/docker.yml

```diff
@@ -7,9 +7,9 @@ tasks:
     - printenv > /tmp/.envfile
     - sed -i 's/PATH.*//' /tmp/.envfile
     - |
-      docker run --rm -i {{.DOCKER_RUN_FLAGS}} \
+      docker run --rm -i --privileged {{.DOCKER_RUN_FLAGS}} \
         --env-file /tmp/.envfile \
-        --volume '{{.PWD}}:{{.PWD}}:Z' \
+        --volume '{{.PWD}}:{{.PWD}}' \
```

Comment:
is the `Z` a typo or... just a magic constant?

Reply:
that's a tag for SELinux, but in order to avoid having the kernel create tags for the thousands of files in the build directory, we use `--privileged` instead, which bypasses the SELinux module instead. This is fine for dev environments but not recommended in production ones